### ISSUE TITLE
Hide / disable parent container when dragging

### ISSUE THREAD

**Issue by ioRekz** [react-dnd/react-dnd#912]
Hello, I'm trying to have this behaviour using react-dnd ![dragndrop](https://thumbs.gfycat.com/LegitimateUniqueEmu-size_restricted.gif)
The problem is for hiding/disabling the whole menu.
I've tried not rendering but html5 dnd need a node to drag
I've tried opacity 0 but then I can't drop on the left part because the menu is still here on top.
I then tried height 0 as in the customdraglayer example (even if the comment says it's there for IE6) but if I drop on the exact same spot I start dragging, onDrop is not called (I supposed it's drop on the dragsource which is on top)
Tried Negative z-index but I can't see my custom drag layer

Any idea ? 

ps : the discord channel need-help is too broad and a message get lost in 10mins

**Comment by aralroca** [comment]
I have the same problem. @ioRekz do you found the way to solve it? Thanks !

**Comment by AlabamaYarrow** [comment]
I was experiencing similar issue, I needed to drag-n-drop element from a popup block and hide the popup while dragging. For some reason any operation on popup (like changing z-index or positioning) terminated the drag. The only working solution I found (quite a hacky one, I'm afraid) was to manually apply css-translate after a timeout in parent component method, which is passed to dragged element via props and called from `beginDrag`. So dragging animation starts before you move container.
```javascript
onBeginDrag() {
    setTimeout(() => {
       this.setState({ movePopupAway: true });
    }, 1);
}
```
And in `render()` method something like:
```javascript
if (this.state.movePopupAway) {
     popupStyle.transform = 'translateX(9999px)';
}
```
